crates/meroctl/src/cli.rs

```diff
@@ -71,14 +71,23 @@ pub struct RootArgs {
 
     #[arg(long, value_name = "FORMAT", default_value_t, value_enum)]
     pub output_format: Format,
+
+    #[arg(long, value_name = "PROTOCOL")]
+    pub protocol: String,
 }
 
 impl RootArgs {
-    pub const fn new(home: Utf8PathBuf, node_name: String, output_format: Format) -> Self {
+    pub const fn new(
+        home: Utf8PathBuf,
+        node_name: String,
+        output_format: Format,
+        protocol: String,
+    ) -> Self {
         Self {
             home,
             node_name,
             output_format,
+            protocol,
```

Comment:
we shouldn't need `--protocol` at the root level, since it's only useful for `context create`